backend/build.gradle

```diff
@@ -38,6 +38,9 @@ dependencies {
 //	Okta dependencies
 	implementation 'com.okta.spring:okta-spring-boot-starter:1.4.0'
 
+// App insights instrumentation
+	implementation 'com.microsoft.azure:applicationinsights-core:2.6.2'
```

Comment:
I observe there is a [Spring Boot Starter](https://mvnrepository.com/artifact/com.microsoft.azure/applicationinsights-spring-boot-starter/2.6.2) but nobody so far can tell me why I should use it instead of just doing what you've done here. That seems worth investigating, unless you already did.

Reply:
I did not, would be worth looking into. The runtime agent seems to be gathering at least some of the metrics we need.